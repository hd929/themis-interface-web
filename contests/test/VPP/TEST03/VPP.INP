8
1 24
2
1 72
2
2
1 92
1 64
1 23
