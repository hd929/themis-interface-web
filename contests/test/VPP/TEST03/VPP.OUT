92 64 23
